### ISSUE TITLE
res.json(null), no response

### ISSUE THREAD

**Issue by calumk** [talentlessguy/tinyhttp#223]
**Describe the bug**

Documentation says `null` is a valid argument https://tinyhttp.v1rtl.site/docs#resjson

However if you pass null to res.json() then no response is sent back to client

```javascript
res.json(null) // doesnt work
res.json({}) // works

res.send(null) // works

res.end(null) // works

```

**How to fix**

the issue is that if `body = null` , it causes an edge case here:

https://github.com/talentlessguy/tinyhttp/blob/a909c9918460eeef4e0ed2509399ed7b46149466/packages/send/src/json.ts#L11-L12

I dont think the check for `typeof  body === string` is required

**Comment by talentlessguy** [comment]
Could you please send a PR with a bugfix? (should be smth as easy as `body || {}`)

currently don't have time to work on tinyhttp :v

**Comment by calumk** [comment]
Done, Code deliberately verbose for clarity.

**Comment by talentlessguy** [comment]
Bugfix landed in 344a8ae79c72a803dba5b34614d5272c432bbf8c, so closing
